charts/lagoon-logging/templates/logs-dispatcher.fluent-conf.configmap.yaml

```diff
@@ -270,7 +270,6 @@ data:
       skip_container_metadata true
       skip_master_url         true
     </filter>
-{{- end }}
```

Comment:
Hey @Schnitzel this might be a mistake because it captures the `index_name` filter block inside the `.Values.openshiftHaproxyLogsCollector.enabled` block. Instead I think that the filter block adding `index_name` should go _after_ the `openshiftHaproxyLogsCollector` block?

Reply:
Oh I think I get it.. `router-logs-e2e` aren't a thing on Kubernetes?